### DISCUSSION TITLE
Please provide instructions on how to use/install  this plugin.

### DISCUSSION THREAD

**Discussion by S2GN** [retrooper/packetevents#867] [Q&A] [answered]
Hello,

I was just notified that I needed to update my version of PacketEvents to +v2.4.0 in support of proper LibsDisguises fuctionality.
(FYI I've never heard of or installed PacketEvents so this appears to be a new dependency)

My questions are:
1- Does PacketEvents replace ProtoclLib?
2 - I run a Velocity Network, do I need to install this on the Velocity Proxy, and all back-end Servers?

Thank you,

ChromaKat

**Comment by booky10** [comment, marked as answer]
1. No, packetevents is a Plugin different to ProtocolLib - they serve the same purpose, but are both completely different
2. You need to install it on the server which runs LibsDisguises

Packetevents is a plugin which can be used by other plugin developers and therefore doesn't do anything on its own.
